File: presentation/src/main/java/by/grodno/vasili/githubsimpleapp/feature/users/UserItem.java
package by.grodno.vasili.githubsimpleapp.feature.users;

/**
 * User model for presentation in list of Users
 */
public class UserItem {
    public String id;
    public String login;
    public String avatarUrl;

    UserItem(String id, String login, String avatarUrl) {
        this.id = id;
        this.login = login;
        this.avatarUrl = avatarUrl;
    }
}

File: presentation/src/main/java/by/grodno/vasili/githubsimpleapp/feature/users/UserItem.kt
package by.grodno.vasili.githubsimpleapp.feature.users

import by.grodno.vasili.domain.model.User

/**
 * User model for presentation layer.
 */
data class UserItem(
        @JvmField var id: String,
        @JvmField var login: String,
        @JvmField var avatarUrl: String
) {
    constructor(user: User) : this(
            id = user.id,
            login = user.login,
            avatarUrl = user.avatarUrl
    )
}
